pygeo/DVGeometry.py

```diff
@@ -2133,6 +2133,81 @@ def writePlot3d(self, fileName):
             """
         self.FFD.writePlot3dCoef(fileName)
 
+    def updatePyGeo(self, geo, outputType, fileName, nRefU=0, nRefV=0):
+        """ Deform a pyGeo object and write to a file of specified type
+        given the (deformed) current state of the FFD object.
+
+        Parameters
+        ----------
+        geo : pyGeo object
+            A pyGeo object containing an initialized object
+        outputType: str
+            Type of output file to be written. Can be `iges` or `tecplot`
+        fileName: str
+            Filename for the output file. Should have no extension, an
+            extension will be added
+        nRefU: int or list of ints
+            Number of spline refinement points to add in the surface B-Spline u-direction.
+            If scalar, it is applied across each surface. If list, the length must match the
+            number of surfaces in the object and corresponding entries are matched with surfaces.
+        nRefV: int or list of ints
+            Number of spline refinement points to add in the surface B-Spline v-direction.
+            If scalar, it is applied across each surface. If list, the length must match the
+            number of surfaces in the object and corresponding entries are matched with surfaces
+        """
+        # Refine Surface -- U-Direction
+        if type(nRefU) is int:
```

Comment:
Type checking should be done using `isinstance()` rather than `type()`

Reply:
Fixed.